packages/docs/docs/guides/5-request-client/1-payment-networks.md

```diff
@@ -5,55 +5,86 @@ keywords: [Request Client, Payment networks]
+**Input data:**
 
-There are different methods available for the requester and payer to agree on the payment status, and that is when Payment networks come into play.
+In certain cases, when transferring a currency amount, the user has the choice to add additional data to the transaction. For example, Ethereum allows the user to add miscellaneous data named *input data* when performing a simple ether transfer.
+In this case, the payment reference is documented here.
 
-Payment networks allow developers to:
-* Detect blockchain-based payment automatically
-* Declare other kinds of payments completion
+**Proxy smart contract:**
 
-Request is blockchain agnostic, payment methods that are suppored for automated payment dection are: Bitcoin, Ether and ERC20. If you miss your favorite blockchain, you can either get in touch with the team or **TODO learn how to do it yourself**
+In this case, the reference is documented through a proxy smart contract.
+This is a smart contract that forwards a currency transfer and stores a reference.
+The currency must be backed by a blockchain with smart contract capabilities.
 
-## Ethereum and ERC20
+### Declarative
 
-Because one Ethereum address is generally used many times to receive and send transactions, we need a way to identify each payment. Request provides two payment channels for ETH and ERC20:
+For this payment network, the request doesn't specify any additional data. This is the request's stakeholders that declare the payments and refunds.
+The payee declares the received payments and the payer declares the received refunds. The balance of the request is the sum of declared payments minus the sum of declared refunds.
+The payee can also declare the sent refunds and the payer the sent payments. These declarations are used only for documentation purposes and aren't taken into consideration to compute the request balance.
 
-### Address-based payment detection
-The payment channel `pn-erc20-address-based` can be used for one-shot address that will only receives payments for a single request.
+This payment network can be used with every currency.
 
-Before using this payment channel, you then have to make sure you create two addresses: one for the payment (or payments if the request is paid in several parts), and one for the refund.
+## Currencies
 
-### Proxy-contract
-This is the most convenient way for most use cases.
+The currencies that are supported for automated payment detection are
+* Bitcoin
+* Ether
+* ERC20
 
-If you opt for the `pn-erc20-proxy-contract` payment channel, the flow for your users will be:
+### Bitcoin
 
-1. The user allows the smart contract to transfer funds from his address, up to a certain amount you choose.
+Request allows to create Bitcoin request with an address based payment network.
+This payment network is sufficient for Bitcoin requests because generating a new address for every inbound BTC transfers is already part of the Bitcoin's good practices.
```

Comment:
```suggestion
This payment network is sufficient for Bitcoin requests because generating a new address for every inbound BTC transfers is already part of Bitcoin's good practices.
```